src/test/groovy/ApmBasePipelineTest.groovy

```diff
@@ -746,6 +747,15 @@ class ApmBasePipelineTest extends DeclarativePipelineTest {
     return jsonSlurper.parseText(jsonText)
   }
 
+  def readNetJSON(params){
```

Comment:
readNetJson? why `Net` in the name 😄

Reply:
It uses `net.sf.json.JSONSerializer` so I wanted a new test function so the tests could benefit from, but I didn't want to change the existing implementation since it might produce some errors in the existing UTs.

I can change it for something else, open for suggestions

Reply:
I'd name it `toJSON` and delegate the implementation details to the internals of the method

Reply:
I'm afraid `toJSON` clashes with some existing calls with a different signature. so I tried to refactor it, but I'm afraid it's not that simple. I just renamed the function with something else most descriptive about what args are allowed.